tests/pyathena/sqlalchemy/test_base.py

```diff
@@ -1796,3 +1796,23 @@ def test_insert_from_select_cte_follows_insert_one(self, engine):
             ).strip()
         )
         assert actual == [(2, "bar")]
+
+    def test_get_view_definition(self, engine):
+        engine, conn = engine
+        insp = sqlalchemy.inspect(engine)
+        actual = insp.get_view_definition(schema=ENV.schema, view_name="v_one_row")
+        assert ([r for r in actual] == [
+            f"CREATE VIEW {ENV.schema}.v_one_row AS",
+            "SELECT number_of_rows",
+            "FROM",
+            f"  {ENV.schema}.one_row",
+        ]
+                )
+
+    def test_get_view_definition_missing_view(self, engine):
+        engine, conn = engine
+        insp = sqlalchemy.inspect(engine)
+        pytest.raises(
+            OperationalError,
```

Comment:
I would like to see NoSuchTableError checked.

Reply:
sure 🙈